src/ArtifactoryUploader/PackageUploader.cs

```diff
@@ -59,11 +67,69 @@ public static async Task UploadPackageToArtifactory(CommonAppSettings appSetting
             Logger.Debug($"UploadPackageToArtifactory():End");
 
             // set the error code
-            if(uploaderKpiData.PackagesNotUploadedDueToError > 0 || uploaderKpiData.PackagesNotExistingInRemoteCache > 0)
+            if (uploaderKpiData.PackagesNotUploadedDueToError > 0 || uploaderKpiData.PackagesNotExistingInRemoteCache > 0)
             {
                 Environment.ExitCode = 2;
                 Logger.Debug("Setting ExitCode to 2");
             }
         }
+
+        private static void DiplayAllSettings(List<Component> m_ComponentsInBOM, CommonAppSettings appSettings)
```

Comment:
did u cover this method in UT

Reply:
yes its covered in PackageUploaderTest.cs